pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"time"
 
+	v1 "github.com/rancher/rancher/pkg/generated/norman/apps/v1"
+
```

Comment:
nit: empty space. In general if you use IDE like goland it will do this. You have to manually check imports to remove it